WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -405,10 +402,10 @@ - (void)layoutControls
     _skipButton.frame = CGRectMake(skipButtonX, skipButtonY, CGRectGetWidth(_skipButton.frame), CGRectGetHeight(_skipButton.frame));
 
     NSArray *viewsToCenter;
-    if (self.blog.hasJetpack) {
+    if (self.blog.jetpack.isInstalled) {
```

Comment:
Considering there's a similar change above: shouldn't this also test against `self.blog.jetpack.isUpdatedToRequiredVersion`?  If that's the case these two conditions should probably wrapped in a single method.